tests/controllers/test_crop_controller.py

```diff
@@ -0,0 +1,98 @@
+import unittest
+import yaml
+import bpy
+import os
+from src.controllers.crop_controller import CropController
+from src.controllers.yaml_reader import YamlReader
+
+
+class CameraControllerTest(unittest.TestCase):
+    def setUp(self):
+        # Set up test environment
+        self.test_file = "tests/test_data.yml"
+        self.test_output = "tests/test_output.png"
+        self.collection = "Test Collection"
+        self.expected_material_name = "Red"
+        self.expected_segmentation_id = 1
+
+    def tearDown(self):
+        # Clean up test environment
+        os.remove(self.test_file)
```

Comment:
Missing teardown of "tests/test_output.png" i.e. self.test_output

Reply:
There is no teardown for "test/test_output.png" as it is not created while checking. I will delete the "tests/test_output.png" in the setup

Reply:
Thank you for the clarification